src/basic/time-util.c

```diff
@@ -459,6 +459,24 @@ int dual_timestamp_deserialize(const char *value, dual_timestamp *t) {
         return 0;
 }
 
+usec_t deserialize_timestamp_value(const char *value, usec_t timestamp) {
+        unsigned long long a;
+
+        assert(value);
+
+        if (!value)
+                return 0;
+
+        if (sscanf(value, "%llu", &a) != 1) {
+                log_debug("Failed to parse finish timestamp value %s.", value);
+                return 0;
```

Comment:
I think this function should just return negative values on error, and 0 on success.  Store the result in `*timestamp`, assuming it becomes a `usec_t *`, which I imagine was the intention of the timestamp parameter?